package-res/resources/web/pentaho/data/filter/IsIn.js

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "./AbstractPropertyFilter",
+  "./_toSpec"
+], function(AbstractPropertyFilter, toSpec) {
+  "use strict";
+
+  /**
+   * @name IsIn
+   * @memberOf pentaho.data.filter
+   * @class
+   * @extends pentaho.data.filter.AbstractPropertyFilter
+   * @amd pentaho/data/filter/IsIn
+   *
+   * @classdesc The `IsIn` class implements a type of filter for a data table {@link pentaho.data.Table}.
```

Comment:
@classdesc A filter that defines the set of items in which the value of a property belongs to a given set.